src/lib/y2firewall/widgets/interfaces_table.rb

```diff
@@ -0,0 +1,115 @@
+# encoding: utf-8
+
+# Copyright (c) [2018] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "cwm"
+require "cwm/table"
+require "y2firewall/ui_state"
+
+module Y2Firewall
+  module Widgets
+    # A table with all {Y2Firewall::Firewalld::Interface}s.
+    class InterfacesTable < ::CWM::Table
+      DEFAULT_ZONE_NAME = "default".freeze
```

Comment:
No i18n here?

Reply:
I am not sure about it because we are not translating zones names.